packages/charts/src/components/legend/legend_icon.tsx

```diff
@@ -28,11 +28,17 @@ const getAdjustedRadius = (radius: number | undefined, strokeWidth: number) => {
 
 /** @internal */
 export const LegendIcon = ({ pointStyle, color }: LegendIconProps) => {
-  const { radius = 4, fill, strokeWidth = 1, stroke, shape = PointShape.Circle, opacity = 1 } = pointStyle?.shape
+  const {
+    radius = 4,
```

Comment:
This is not needed, this default is handled in the logic below.

```suggestion
    radius,
```

Reply:
57d1684 thank you!